firefox_puppeteer/api/appinfo.py

```diff
@@ -6,17 +6,38 @@
 
 
 class AppInfo(BaseLib):
+    """This class provides access to various attributes of AppInfo.
+
+    For more details on AppInfo, visit:
+    https://developer.mozilla.org/en-US/docs/Mozilla/QA/Mozmill_tests/Shared_Modules/UtilsAPI/appInfo
+    """
+
+    def __getattr__(self, attr):
+        with self.marionette.using_context('chrome'):
+            value = self.marionette.execute_script("""
+              return Services.appinfo[arguments[0]];
+            """, script_args=[attr])
+
+            if value is None:
+                return value
```

Comment:
Actually, i am backing down for a few days from here. I will learn git first. I was making changes here in the git text editor itself. 
And I don't have access to IRC. I work on an institutional network. So we need to work out something else.